assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -141,6 +130,14 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 					) }
 				</Row>
 			</div>
+
+			<div className="googlesitekit-widget-area-widgets">
+				{ inactiveWidgets.map( ( widget ) => (
+					<div className={ HIDDEN_CLASS } key={ widget.slug }>
+						<widget.Component />
+					</div>
+				) ) }
+			</div>
```

Comment:
The whole `div` here should be hidden, it doesn't need any other class on it. We don't need to render an extra div around each widget.

Reply:
Updated.